HISTORY

```diff
@@ -1,6 +1,13 @@
 VDR Plugin 'vnsiserver' Revision History
 ----------------------------------------
 
+2015-04-06: Version 1.3.1
```

Comment:
no need to bump version, we have not released 1.3.0 yet
https://github.com/FernetMenta/vdr-plugin-vnsiserver/releases